compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/BeamKeyExtractor.java

```diff
@@ -26,7 +26,8 @@
   @Override
   public Object extractKey(final Object element) {
     if (element instanceof KV) {
-      return ((KV) element).getKey();
+      final Object key = ((KV) element).getKey();
+      return key == null ? 0 : key;
```

Comment:
Leave a comment here that Beam allows KV(null, value)?

Reply:
Thank you, I will leave a comment.